composer.json

```diff
@@ -23,7 +23,10 @@
         "matthiasnoback/symfony-dependency-injection-test": "~0.6",
         "matthiasnoback/symfony-config-test": "^1.3.1",
         "phpunit/php-code-coverage": "@stable",
-        "sonata-project/doctrine-phpcr-admin-bundle": "^1.1",
+        "sonata-project/admin-bundle": ">=2.2.12",
+        "sonata-project/core-bundle": ">=2.2.3",
+        "sonata-project/block-bundle": ">=2.2.8",
```

Comment:
Yep, I noticed! Should I change `sonata-project/admin-bundle` or `sonata-project/core-bundle` as well? I didn’t want to be to strict as I don’t know the recent development of the packages.